packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -256,6 +305,88 @@ export class ARRenderer extends EventDispatcher<
     this.dispatchEvent({type: 'status', status: ARStatus.SESSION_STARTED});
   }
 
+  private intersectObjects(controller: XRTargetRaySpace) {
+    // Do not highlight in mobile-ar
+    if (controller.userData.targetRayMode === 'screen')
+      return;
+
+    // Do not highlight when already selected
+    if (controller.userData.selected !== undefined)
+      return;
```

Comment:
I noticed the frame rate is quite low when no object is selected, but when I have an object in my grip the frame rate is fine. I believe this early return is why: the raycast against the whole GLB is slow. We should replace this with a bounding box cube, maybe the placement box, to fix this.

Reply:
Fixed, though haven't updated from hit plane to box yet.

Reply:
Thanks!